playbooks/roles/rabbitmq/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - role: newrelic_infra
```

Comment:
Should also update these either in [rabbitmq/defaults/main.yml](https://github.com/open-craft/ansible-playbooks/blob/master/playbooks/roles/rabbitmq/defaults/main.yml#L24-L25) or in the private vars, since that's the version that's now running on stage:

```
ERLANG_VERSION: "1:23.1"   # not sure about this format?
RABBITMQ_VERSION: "3.8.9"
```

Reply:
That's part of the reason we're getting dependency issues on stage; I think we'll need to re-pin back... Will have a play and see how it goes.

Reply:
As discussed, we don't want to deal with upgrading rabbitmq, so the existing pinned versions can stay. :)